### ISSUE TITLE
Disable elements don't need to meet contrast ratio requirements

### ISSUE THREAD

**Issue by cnamal** [microsoft/accessibility-insights-web#981]
Go to https://developer.microsoft.com/en-us/fabric#/controls/web/combobox
Run Accessibility insights

-> A11y insights reports a color contrast issue
![image](https://user-images.githubusercontent.com/9625284/62028642-0a4bab80-b1e1-11e9-9797-e5339aed387f.png)

I opened a bug on [Office UI Fabric](https://github.com/OfficeDev/office-ui-fabric-react/issues/9909), which pointed me to the [following exception](https://www.w3.org/WAI/GL/low-vision-a11y-tf/wiki/Contrast_(Minimum)#SC_Text):

> Inactive: Disabled or otherwise inactive user interface components are exempt from this requirement. In all other states (including when the page is scrolled) the contrast requirement is applicable.

It therefore seems like Accessibility Insights shouldn't report an issue when a label associated to a disabled input doesn't meet contrast ratio requirements.

**Comment by iamrafan** [comment]
@cnamal , thanks for filing the issue. 

Yes, Accessibility Insights shouldn't be reporting this issue. We filed it (https://github.com/dequelabs/axe-core/issues/1731) with axe-core and will update you once we fix it.

**Comment by ferBonnin** [comment]
@flyingUnderTheRadar can you take a look a the bug opened in axe-core?

**Comment by ferBonnin** [comment]
dequelabs/axe-core#1731 is now closed

**Comment by iamrafan** [comment]
The fix for this issues is part of axe-core 3.4.0 release

**Comment by xugao** [comment]
@ferBonnin  @iamrafan  this bug is still reproing on https://developer.microsoft.com/en-us/fabric#/controls/web/combobox site. and it seems to only be fixed if the disabled element is an input.

is the axe-core not included in the latest extension yet?

**Comment by ferBonnin** [comment]
Hi @xugao we have not consumed the 3.4 axe-core release yet, but this is in our backlog. We will update this issue once we have updated the rule set.

Thanks!

**Comment by ferBonnin** [comment]
@iamrafan we have updated to axe-core 3.5.1, can you check this issue?

**Comment by iamrafan** [comment]
Hi @ferBonnin , @xugao: The issue we initially filed with axe-core has been fixed, but looks like the way this disabled control is labelled in the Office Fabric page has been changed from using ```<label>``` element & ```for``` attribute to use ```aria-labelledby``` attribute.

axe-core fails such labeling, so I filed issue https://github.com/dequelabs/axe-core/issues/2090 with axe-core.

**Comment by ferBonnin** [comment]
@iamrafan this should now be addressed in Accessibility Insights for Web Version 2.21.1 | axe-core 4.0.1; could you please confirm?

**Comment by iamrafan** [comment]
@ferBonnin Yes, this issue has been fixed. Here is the demo page: https://codepen.io/iamrafan/pen/dyMzzmo

**Comment by ferBonnin** [comment]
hey @cnamal this has now been addressed

**Comment by craig-o-curtis** [comment]
I'm getting this same issue running accessibility insights - failing contrast issue on a disabled combobox... :(

**Comment by ferBonnin** [comment]
@craig-o-curtis thanks for reaching out, can you please share with us a repro page or codepen? we would love to investigate this more!
